src/arrayReverse.test.js

```diff
@@ -8,13 +8,27 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(arrayReverse(['a2'])).toBeInstanceOf(Array);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse([])).toEqual([]);
```

Comment:
Suggestion to change [ ]: empty string in array will be [' ']

Reply:
empty string = ''
empty array = []
array with empty string != empty array